src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -59,6 +59,10 @@ const ERROR_VISUALIZATION_SIZE       : (f32,f32) = visualization::container::DEF
 
 const VISUALIZATION_OFFSET_Y         : f32       = -120.0;
 
+/// A type of unresolved methods. We filter them out, because they are not helpful from the UX
+/// perspective.
```

Comment:
This comment should be better - not "they are not helpful from the UX perspective". Rather - we don't want to treat them as types fo port coloring etc. Such comments should describe semantically why such a change is made.